tests/functional/modules/test_zos_apf_func.py

```diff
@@ -200,65 +178,106 @@ def test_add_del_persist(ansible_zos_module):
+        cmdStr = "dtouch -tseq {0}".format(prstds)
+        hosts.all.shell(cmd=cmdStr)
+        test_info['persistent']['data_set_name'] = prstds
     results = hosts.all.zos_apf(**test_info)
-    pprint(vars(results))
     for result in results.contacted.values():
         assert result.get("rc") == 0
     add_exptd = add_expected.format(test_info['library'], test_info['volume'])
     add_exptd = add_exptd.replace(" ", "")
     cmdStr = "cat \"//'{0}'\" ".format(test_info['persistent']['data_set_name'])
-    actual = run_shell_cmd(hosts, cmdStr).replace(" ", "")
+    results = hosts.all.shell(cmd=cmdStr)
+    for result in results.contacted.values():
+        actual = result.get("stdout")
+    actual = actual.replace(" ", "")
     assert actual == add_exptd
     test_info['state'] = 'absent'
     results = hosts.all.zos_apf(**test_info)
-    pprint(vars(results))
     for result in results.contacted.values():
         assert result.get("rc") == 0
     del_exptd = del_expected.replace(" ", "")
     cmdStr = "cat \"//'{0}'\" ".format(test_info['persistent']['data_set_name'])
-    actual = run_shell_cmd(hosts, cmdStr).replace(" ", "")
+    results = hosts.all.shell(cmd=cmdStr)
+    for result in results.contacted.values():
+        actual = result.get("stdout")
+    actual = actual.replace(" ", "")
     assert actual == del_exptd
     clean_test_env(hosts, test_info)
 
 
-def test_batch_add_del(ansible_zos_module):
-    hosts = ansible_zos_module
-    test_info = TEST_INFO['test_batch_add_del']
-    for item in test_info['batch']:
-        set_test_env(hosts, item)
-    test_info['persistent']['data_set_name'] = persistds_create(hosts)
-    results = hosts.all.zos_apf(**test_info)
-    pprint(vars(results))
-    for result in results.contacted.values():
-        assert result.get("rc") == 0
-    add_exptd = add_batch_expected.format(test_info['batch'][0]['library'], test_info['batch'][0]['volume'],
-                                          test_info['batch'][1]['library'], test_info['batch'][1]['volume'],
-                                          test_info['batch'][2]['library'], test_info['batch'][2]['volume'])
-    add_exptd = add_exptd.replace(" ", "")
-    cmdStr = "cat \"//'{0}'\" ".format(test_info['persistent']['data_set_name'])
-    actual = run_shell_cmd(hosts, cmdStr).replace(" ", "")
-    assert actual == add_exptd
-    test_info['state'] = 'absent'
-    results = hosts.all.zos_apf(**test_info)
-    pprint(vars(results))
-    for result in results.contacted.values():
-        assert result.get("rc") == 0
-    del_exptd = del_expected.replace(" ", "")
-    cmdStr = "cat \"//'{0}'\" ".format(test_info['persistent']['data_set_name'])
-    actual = run_shell_cmd(hosts, cmdStr).replace(" ", "")
-    assert actual == del_exptd
-    for item in test_info['batch']:
-        clean_test_env(hosts, item)
-    persistds_delele(hosts, test_info['persistent']['data_set_name'])
+#def test_batch_add_del(ansible_zos_module):
```

Comment:
Test comment related to [issue](https://github.com/ansible-collections/ibm_zos_core/issues/726)

Reply:
Good points, seems safe to resolve , we have an open issue for it for 1.3 so we will uncomment it.